builder/overlay-rootfs/etc/s6/init/init-stage1

```diff
@@ -36,6 +36,8 @@ exec -c --
 s6-envdir /var/run/s6/env-stage1
 exec --
 
+backtick -n S6_CMD_ARG0 { printcontenv S6_CMD_ARG0 }
+import -s -u S6_CMD_ARG0
```

Comment:
@redbaron Can you change this to use `importas`? IIRC, we opted to use `importas` over `import` because ImageMagick has a conflicting `import` program.